plaid/webhooks.go

```diff
@@ -0,0 +1,53 @@
+package plaid
+
+import (
+	"encoding/json"
+	"errors"
+)
+
+type WebhookVerificationKey struct {
+	Alg       string `json:"alg"`
+	CreatedAt int64  `json:"created_at"`
+	Crv       string `json:"crv"`
+	ExpiredAt int64  `json:"expired_at"`
+	Kid       string `json:"kid"`
+	Kty       string `json:"kty"`
+	Use       string `json:"use"`
+	X         string `json:"x"`
+	Y         string `json:"y"`
+}
+
+type GetWebhookVerificationKeyResponse struct {
+	APIResponse
+	Key WebhookVerificationKey `json:"key"`
+}
+
+type getWebhookVerificationKeyRequest struct {
+	ClientID string `json:"client_id"`
+	Secret   string `json:"secret"`
+	KeyID    string `json:"key_id"`
+}
+
+// GetWebhookVerificationKey retrieves the verification key for a given webhook verification key ID
+// See https://plaid.com/docs/#webhook-verification.
```

Comment:
⚠️ is this link correct? it just takes me to the docs root.

Reply:
the docs arent merged yet, they will be soon